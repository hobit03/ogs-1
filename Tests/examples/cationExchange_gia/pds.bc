GeoSys-BC: Boundary Conditions ------------------------------------------------
#BOUNDARY_CONDITION
 $PCS_TYPE
  GROUNDWATER_FLOW
 $PRIMARY_VARIABLE 
  HEAD
 $GEO_TYPE
  POINT POINT1
 $DIS_TYPE
  CONSTANT 2.00000

#BOUNDARY_CONDITION
 $PCS_TYPE
  REACT_GIA
 $PRIMARY_VARIABLE 
  Ca
 $GEO_TYPE
  POINT POINT0
 $DIS_TYPE
  CONSTANT 0.600e-3

#BOUNDARY_CONDITION
 $PCS_TYPE
  REACT_GIA
 $PRIMARY_VARIABLE 
  Cl
 $GEO_TYPE
  POINT POINT0
 $DIS_TYPE
  CONSTANT 1.20e-3

#BOUNDARY_CONDITION
 $PCS_TYPE
  REACT_GIA
 $PRIMARY_VARIABLE 
  K
 $GEO_TYPE
  POINT POINT0
 $DIS_TYPE
  CONSTANT 1E-30

#BOUNDARY_CONDITION
 $PCS_TYPE
  REACT_GIA
 $PRIMARY_VARIABLE 
  Na
 $GEO_TYPE
  POINT POINT0
 $DIS_TYPE
  CONSTANT 1E-30

#BOUNDARY_CONDITION
 $PCS_TYPE
  REACT_GIA
 $PRIMARY_VARIABLE 
  CaX2
 $GEO_TYPE
  POINT POINT0
 $DIS_TYPE
  CONSTANT 1E-30

#BOUNDARY_CONDITION
 $PCS_TYPE
  REACT_GIA
 $PRIMARY_VARIABLE 
  NaX
 $GEO_TYPE
  POINT POINT0
 $DIS_TYPE
  CONSTANT 1E-30

#BOUNDARY_CONDITION
 $PCS_TYPE
  REACT_GIA
 $PRIMARY_VARIABLE 
  KX
 $GEO_TYPE
  POINT POINT0
 $DIS_TYPE
  CONSTANT 1E-30

#BOUNDARY_CONDITION
 $PCS_TYPE
  REACT_GIA
 $PRIMARY_VARIABLE 
  X
 $GEO_TYPE
  POINT POINT0
 $DIS_TYPE
  CONSTANT 1E-30
#STOP


2.000000228


#BOUNDARY_CONDITION
 $PCS_TYPE
  GROUNDWATER_FLOW
 $PRIMARY_VARIABLE 
  HEAD
 $GEO_TYPE
  POINT POINT1
 $DIS_TYPE
  CONSTANT 2.0


#BOUNDARY_CONDITION
 $PCS_TYPE
  GROUNDWATER_FLOW
 $PRIMARY_VARIABLE 
  HEAD
 $GEO_TYPE
  POINT POINT0
 $DIS_TYPE
  CONSTANT 0.0197

