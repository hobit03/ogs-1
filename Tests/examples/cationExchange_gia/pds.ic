GeoSys-IC  Initial Conditions ------------------------------------------------
#INITIAL_CONDITION
 $PCS_TYPE
  GROUNDWATER_FLOW
 $PRIMARY_VARIABLE 
  HEAD
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 2.0

#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA
 $PRIMARY_VARIABLE 
  Ca
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 1E-30

#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA
 $PRIMARY_VARIABLE 
  Cl
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 1E-30

#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA
 $PRIMARY_VARIABLE 
  K
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 2.0e-4

#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA
 $PRIMARY_VARIABLE 
  Na
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 1.0e-3

#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA
 $PRIMARY_VARIABLE 
  CaX2
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 1E-30

#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA
 $PRIMARY_VARIABLE 
  NaX
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 5.493e-4

#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA
 $PRIMARY_VARIABLE 
  KX
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 5.507e-4

#STOP


#INITIAL_CONDITION
 $PCS_TYPE
  REACT_GIA
 $PRIMARY_VARIABLE 
  X
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 1e-30

#INITIAL_CONDITION
 $PCS_TYPE
  GROUNDWATER_FLOW
 $PRIMARY_VARIABLE 
  HEAD
 $GEO_TYPE
  DOMAIN
 $DIS_TYPE
  CONSTANT 2.0

