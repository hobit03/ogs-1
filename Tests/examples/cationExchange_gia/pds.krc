GeoSys Reactions
#KINREACTIONDATA      
 $SOLVER_TYPE
	1                ; Solver type
 $RELATIVE_ERROR	
	1.e-6		 ; rel. Error
 $MIN_TIMESTEP	
	1.e-6		; min Timestep
 $INITIAL_TIMESTEP	
	1.e-4       ; init Timestep
 $BACTERIACAPACITY
  100.0
 $NO_REACTIONS
   POLYLINE PLY_BC_LEFT
 $ACTIVITY_MODEL
    2 ; constant activity coefficient = 0, DH = 1, Davies = 2


#REACTION ; 
 $NAME
   R1 ;
 $TYPE
   SORP_EQ_REACT;
 $EQUATION
   1 Na + 1 X = 1 NaX; 
 $EQ_CONSTANT
 1.0           ; K values
 
#REACTION ; 
 $NAME
   R2 ;
 $TYPE
   SORP_EQ_REACT;
 $EQUATION
   1 K + 1 X = 1 KX;  
 $EQ_CONSTANT
 5.011872336           ; K values
 
#REACTION ; 
 $NAME
   R3 ;
 $TYPE
   SORP_EQ_REACT;
 $EQUATION
   0.5 Ca + 1 X = 0.5 CaX2;  
 $EQ_CONSTANT
 6.309573445           ; K values
 
#STOP



#REACTION ; 
 $NAME
   R1 ;
 $TYPE
   SORP_EQ_REACT;
 $EQUATION
   1 NaX = 1 Na + 1 X; 
 $EQ_CONSTANT
 1.0           ; K values
 
#REACTION ; 
 $NAME
   R2 ;
 $TYPE
   SORP_EQ_REACT;
 $EQUATION
   1 KX = 1 K + 1 X;  
 $EQ_CONSTANT
 5.011872336           ; K values
 
#REACTION ; 
 $NAME
   R3 ;
 $TYPE
   SORP_EQ_REACT;
 $EQUATION
   1 CaX2 = 1 Ca + 2 X;  
 $EQ_CONSTANT
 6.309573445           ; K values


#REACTION ; 
 $NAME
   R1 ;
 $TYPE
   SORP_EQ_REACT;
 $EQUATION
   1 Na + 1 X = 1 NaX; 
 $EQ_CONSTANT
 1.0           ; K values
 
#REACTION ; 
 $NAME
   R2 ;
 $TYPE
   SORP_EQ_REACT;
 $EQUATION
   1 K + 1 X = 1 KX;  
 $EQ_CONSTANT
 5.011872336           ; K values
 
#REACTION ; 
 $NAME
   R3 ;
 $TYPE
   SORP_EQ_REACT;
 $EQUATION
    Ca + 2 X = 1 CaX2;  
 $EQ_CONSTANT
 6.309573445           ; K values





 
