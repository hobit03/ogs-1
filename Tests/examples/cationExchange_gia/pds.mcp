#COMPONENT_PROPERTIES  ; comp1
 $NAME
  Ca ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $CHARGE
  +2.0
 $COMP_TYPE
  AQ_PHASE_COMP

#COMPONENT_PROPERTIES  ; comp2
 $NAME  ; 
  Cl ; 
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $CHARGE
  -1.0
 $COMP_TYPE
  AQ_PHASE_COMP

#COMPONENT_PROPERTIES  ; comp3
  $NAME
  K ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $CHARGE
  +1.0
 $COMP_TYPE
  AQ_PHASE_COMP

#COMPONENT_PROPERTIES  ; comp4
 $NAME
  Na ;
 $MOBILE
  1; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $CHARGE
  +1.0
 $COMP_TYPE
  AQ_PHASE_COMP

#COMPONENT_PROPERTIES  ; comp5
 $NAME
  CaX2 ;
 $MOBILE
  0; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $CHARGE
  0.0
 $COMP_TYPE
  SORPTION_COMP

#COMPONENT_PROPERTIES  ; comp6
 $NAME
  NaX ;
 $MOBILE
  0; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $CHARGE
  0.0
 $COMP_TYPE
  SORPTION_COMP

#COMPONENT_PROPERTIES  ; comp7
 $NAME
  KX ;
 $MOBILE
  0; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $CHARGE
  0.0
 $COMP_TYPE
  SORPTION_COMP

#STOP



#COMPONENT_PROPERTIES  ; comp8
 $NAME
  X ;
 $MOBILE
  0; MOBILE-Flag: 0=imMOBILEe, 1=MOBILEe/transported
 $CHARGE
  -1.0
 $COMP_TYPE
  SORPTION_COMP