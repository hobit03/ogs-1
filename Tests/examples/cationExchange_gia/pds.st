GeoSys-ST: Source Terms ------------------------------------------------
#SOURCE_TERM
 $PCS_TYPE
  GROUNDWATER_FLOW
 $PRIMARY_VARIABLE 
  HEAD
 $GEO_TYPE
  POINT POINT0
 $DIS_TYPE
  CONSTANT -2.78e-006
#STOP
	1.3888888e-006   
  CONSTANT  2.7777777e-006
