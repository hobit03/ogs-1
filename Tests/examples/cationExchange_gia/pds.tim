GeoSys-TIM: Timestepping ------------------------------------------------
#TIME_STEPPING
 $PCS_TYPE
  GROUNDWATER_FLOW
 $TIME_STEPS
   480  180.0
 $TIME_END
  86400.0
 $TIME_START
  0.0
#TIME_STEPPING
 $PCS_TYPE
  REACT_GIA
 $TIME_STEPS
   480  180.0
 $TIME_END
  86400.0
 $TIME_START
  0.0
#STOP


#TIME_STEPPING
 $PCS_TYPE
  GROUNDWATER_FLOW
 $TIME_STEPS
   1    20.0
   1    86380.0
 $TIME_END
  86400.0
 $TIME_START
  0.0
#TIME_STEPPING
 $PCS_TYPE
  REACT_GIA
 $TIME_END
  86400.0
 $TIME_START
  0.0
$TIME_CONTROL
   NEWTON_ADAPTIVE
 MAX_TIME_STEP
     1000
 MIN_TIME_STEP
     20.0
 ITER_TIMES_AND_MULTIPLYER
   4
   0    1.6
   7    1.0
   15   0.5
   35   0.25 




