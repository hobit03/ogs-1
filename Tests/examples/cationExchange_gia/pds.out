GeoSys-OUT: Output ------------------------------------------------
#OUTPUT // profile
 $NOD_VALUES
  HEAD
  Ca
  Cl
  K
  Na
  CaX2
  NaX
  KX
 $ELE_VALUES
  VELOCITY
 $GEO_TYPE
  DOMAIN
 $DAT_TYPE
  PVD
 $TIM_TYPE
  STEPS 10
#STOP



#OUTPUT // profile
 $NOD_VALUES
  Ca
  Cl
  K
  Na
  CaX2
  NaX
  KX
 $GEO_TYPE
  POINT  POINT2
 $DAT_TYPE
  PVD
#OUTPUT // profile
 $NOD_VALUES
  HEAD
  Ca
  Cl
  K
  Na
  CaX2
  NaX
  KX
 $GEO_TYPE
  POLYLINE  OUT_LINE
 $DAT_TYPE
  PVD
 $TIM_TYPE
  STEPS 1


 $TIM_TYPE
   0.0
   180.0
   360.0
   8640.0
   86400.0

